resume.md

```diff
@@ -32,26 +32,23 @@ Python, Git, Linux, PHP, Javascript, relational databases (MySQL, PostgreSQL), a
 
 # Experience
 
-\cvevent{2019 - July 2023}{Developer}{Technology Advisors}{
-  {Coded custom features requested by clients using best practices defined by CRM vendors.},
-  {Debugging code when custom features no longer function as expected after CRM upgrades.},
-  {Wrote MySQL scripts for "advanced" reports requested by clients which cannot be built through the CRM's UI.},
-  {Maintained CRM developer certifications when required (SugarCRM, Creatio CRM).},
+\cvevent{2019 - July 2023}{Software Engineer}{Technology Advisors}{
+  {Coded custom features requested by clients using best practices defined by upstream vendors.},
+  {Debugged breakages in custom features caused by dependency upgrades.},
+  {Wrote MySQL scripts for custom reports requested by clients.},
+  {Maintained CRM developer certifications (SugarCRM, Creatio CRM).},
   {Implemented data migrations and maintined data integrations via in-house built ETL application (\href{https://www.starfishetl.com/}{Starfish ETL}).},
-  {Responsible for all on-premise installations of client CRMs including consulting, estimates, application upgrades, stack upgrades, and support.},
-  {Contributed to company blogs.},
-  {Contributed to internal wiki.}
+  {Provided consulting, estimates, application upgrades, stack upgrades, and support for clients' on-premise deployments.},
+  {Authored content for company blogs and internal wiki.}
 }
 
-\cvevent{July 2017 - 2019}{Support}{Technology Advisors}{
-  {Responsible for initial reponse for all incoming support cases.},
-  {Responsible for driving support cases to completion with follow up questions, updates, and checking in with internal resources.},
-  {Built custom sections and pages within the CRMs' UI.},
```

Comment:
I agree the way I described this wasn't clear. These UI changes didn't involve any coding and I guess that was what I was trying to get across. It was literally drag n' drop in the administration section of the CRM.

Reply:
If it was "just" clicking and dragging, I would just omit it :) But I think there's more to it. Would you call it more "interface design" that was the important part?

Reply:
I guess interface design would be stretching the truth a bit. When I got assigned this kind of task, it was usually something along the lines of... "They need a new section.. give it _this_ name. Here is a list of fields and what their lables and types should be. Most of the time it didn't specify what order the fields should be. I would usually just arrange them as similarly as possible to the out-of-the-box sections.

Reply:
>I guess interface design would be stretching the truth a bit. When I got assigned this kind of task, it was usually something along the lines of... "They need a new section.. give it this name. Here is a list of fields and what their lables and types should be. Most of the time it didn't specify what order the fields should be. I would usually just arrange them as similarly as possible to the out-of-the-box sections.

Good call, in that case I think this should be omitted to make space for something more awesome :)